eiffel-vocabulary/EiffelIssueVerifiedEvent.md

```diff
@@ -206,6 +192,7 @@ __Description:__ The encrypted digest. The cryptographic hash function and the d
 ## Version History
 | Version   | Introduced in                                          | Changes                                 |
 | --------- | ------------------------------------------------------ | --------------------------------------- |
+| 1.1.1     | Current version | Replaced data.issues with links
```

Comment:
This ought to be 2.0.0. Required objects have been removed from the event, which means that an outdated consumer will not be able to read it. 

Designating versions is not quite as straight-forward as for an API, though. We should define precise rules to make it easier (but not in this PR, of course).

Reply:
Agreed